x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/security/authz/privilege/ClusterPrivilegeResolver.java

```diff
@@ -153,7 +153,6 @@ public class ClusterPrivilegeResolver {
         RemoteClusterService.REMOTE_CLUSTER_HANDSHAKE_ACTION_NAME,
         RemoteClusterNodesAction.NAME
     );
-    
```

Comment:
is this change needed?

Reply:
CI was complaining about extra spaces at the end of this line. I restored the line as it is in main.